src/app/components/pages/home.tsx

```
@@ -0,0 +1,234 @@
+                </Button>
+                <Button
+                  href="https://forums.runtipi.io"
+                  className="betterhover:hover:bg-gray-700 betterhover:dark:hover:bg-gray-300 w-full bg-black px-4 text-white dark:bg-white dark:text-black"
+                >
+                  Join our forums
+                  <IconMessage className="ml-2 size-6" />
+                </Button>
+              </>
+            }
+          >
+            Runtipi's Discord server is the heart of our community, and we're
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)